includes/wikia/services/UserStatsService.class.php

```diff
@@ -6,34 +6,22 @@ class UserStatsService extends WikiaModel {
 
 	private $userId;
 	private $user;
+	private $wikiId;
 	private $optionsAllWikis;
```

Comment:
Cleanup idea: I think mcache is quick enough and I don't see a reason of aggregating optionsAllWikis again in the object property, what would you say about removing this field? It would simplify this class a lot.